packages/web/src/algorithms/types.ts

```diff
@@ -81,7 +83,7 @@ export interface PcrPrimerChange {
   substitutions: NucleotideSubstitution[]
 }
 
-export interface SubstitutionsWithPrimers extends SubstitutionsWithAminoacids {
+export interface SubstitutionsWithPrimers extends NucleotideSubstitution {
   pcrPrimersChanged: PcrPrimer[]
 }
```

Comment:
The resulting substitutions array will no longer contain `aaSubstitutions` which breaks a few things. See the other comment with the description of the breakage.